data/scannet/batch_load_scannet_data.py

```diff
@@ -16,14 +16,17 @@
 from load_scannet_data import export
 from os import path as osp
 
-SCANNET_DIR = 'scans'
 DONOTCARE_CLASS_IDS = np.array([])
 OBJ_CLASS_IDS = np.array(
     [3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14, 16, 24, 28, 33, 34, 36, 39])
 
 
-def export_one_scan(scan_name, output_filename_prefix, max_num_point,
-                    label_map_file, scannet_dir):
+def export_one_scan(scan_name,
+                    output_filename_prefix,
+                    max_num_point,
+                    label_map_file,
+                    scannet_dir,
+                    is_train=True):
```

Comment:
Use `test_mode` or something to test whether it is test instead of train to be consistent with other judgments? Can simply change the criterion below to 'if not xxx'.

Reply:
Change to test_mode.

Reply:
Change to test_mode.